conda_libmamba_solver/solver.py

```diff
@@ -364,7 +364,21 @@ def _solve_attempt(
         tasks = self._specs_to_tasks(in_state, out_state)
         for (task_name, task_type), specs in tasks.items():
             log.debug("Adding task %s with specs %s", task_name, specs)
-            self.solver.add_jobs(specs, task_type)
+            try:
+                self.solver.add_jobs(specs, task_type)
+            except RuntimeError as exc:
+                if mamba_version() == "1.5.0":
+                    for spec in specs:
+                        if spec in str(exc):
+                            break
+                    else:
+                        spec = f"One of {specs}"
+                    msg = (
+                        "This is a bug in libmamba 1.5.0. 'defaults' (and 'pkgs/*') cannot be "
+                        "used in channel::pkg_name specs. Please use -c defaults instead."
+                    )
```

Comment:
While technically correct, is this really something we want to bubble up to users?

```suggestion
                    msg = (
                        "This is a bug in libmamba 1.5.0 when using 'defaults::"
                        "<spec>' or 'pkgs/main::<spec>'. Please use '-c "
                        " defaults' instead."
                    )
```

Is `pkgs/main::<spec>` correct?

Reply:
```suggestion
                    msg = (
                        "This is a bug in libmamba 1.5.0 when using 'defaults::"
                        "<spec>' or 'pkgs/main::<spec>'. Please use '-c "
                        "defaults' instead."
                    )
```